pkg/operator2/oauthclients.go

```diff
@@ -0,0 +1,78 @@
+package operator2
+
+import (
+	"crypto/sha256"
+	"reflect"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	oauthv1 "github.com/openshift/api/oauth/v1"
+	routev1 "github.com/openshift/api/route/v1"
+	oauthdiscovery "github.com/openshift/library-go/pkg/oauth/oauthdiscovery"
+)
+
+func defaultBrowserOAuthClient(route *routev1.Route) *oauthv1.OAuthClient {
+	return &oauthv1.OAuthClient{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: oauthBrowserClientName,
+		},
+		Secret:       randomString(sha256.BlockSize),
```

Comment:
I'm not entirely sure about the requirements for this secret so this might be wrong